### PR TITLE
Add JAVA12 preprocessor configuration

### PR DESCRIPTION
**Pull request by keithc-ca**
[ci skip] new configuration is not yet used

### CODE REVIEW

File: jcl/jpp_configuration.xml [hunk 1]
```diff
@@ -244,6 +244,33 @@
...
+		<classpathentry kind="src" path="src/openj9.cuda/share/classes"/>
+		<classpathentry kind="src" path="src/openj9.dataaccess/share/classes"/>
+		<classpathentry kind="src" path="src/openj9.dtfj/share/classes"/>
+		<classpathentry kind="src" path="src/openj9.dtfjview/share/classes"/>
+		<classpathentry kind="src" path="src/openj9.gpu/share/classes"/>
+		<classpathentry kind="src" path="src/openj9.sharedclasses/share/classes"/>
+		<classpathentry kind="src" path="src/openj9.traceformat/share/classes"/>
+		<classpathentry kind="src" path="src/openj9.zosconditionhandling/share/classes"/>
+		<classpathentry kind="lib" path="/binaries/common/ibm/ibmjzos.jar"/>
+		<classpathentry kind="lib" path="/binaries/vm/third/rt-compressed.sunJava11.jar"/>
```
Comment (pshipton) [inline jcl/jpp_configuration.xml]: Did you want to change this, or wait until there is a reason?

**Comment by pshipton** [comment]
Re the [ci skip], although its not used, I would run a build just to ensure the additions to the file didn't break it and everything else can still build.

File: jcl/jpp_configuration.xml (hunk 1 above)
Comment (keithc-ca) [reply jcl/jpp_configuration.xml]: I don't think we should create java12.jar until there's a good reason.

**Comment by pshipton** [comment]
jenkins compile plinux jdk8

**Comment by pshipton** [comment]
Ah right, we need omr to promote first.

**Comment by pshipton** [comment]
jenkins compile plinux jdk8

**Comment by pshipton** [comment]
eclipse ci is down

**Pull request merged**